### ISSUE TITLE
Protocol: 'Booking a tech day'

### ISSUE THREAD

**Issue by Adimote** [sourcebots/volunteer-docs#13]

**Comment by trickeydan** [comment]
@antonnikitin97 I think it would be good to document the process for booking Zepler Level 2 here.

**Comment by antonnikitin97** [comment]
Yep, I agree. I can also document how to book 25 if (for whatever reason) Level 2 isn't available :)

**Comment by RealOrangeOne** [comment]
If we're doing booking guides, be careful not to include things like email addresses, and i'd be weary of including names too.
